functions/package.json

```diff
@@ -7,7 +7,11 @@
     "shell": "firebase experimental:functions:shell",
     "start": "npm run shell",
     "deploy": "firebase deploy --only functions",
-    "logs": "firebase functions:log"
+    "logs": "firebase functions:log",
+    "firebase:save:keys": "npm run firebase:upload:env:dev && npm run firebase:upload:env:stage && npm run firebase:upload:env:live",
```

Comment:
I'd like to avoid adding npm scripts in `/functions/package.json` directory since you have to be in `/functions` to run them. Everything else is run from the project root, but if you try that here node will look in the root `/package.json` and return an error.

Reply:
Another reason I'm hesitant to add a script here: I really want users to understand exactly what's happening "behind the scenes" in firefly. Since updating keys should be rare (unlike a deploy) I don't mind making them run a script if it forces them to see what's actually happening.

Open to discussion on all this though!

Reply:
> Only disagreement: I actually did end up running these commands a few times, especially as I played with the test and prod stripe environments. However, I think that's the sort of first step that devs using this template should learn to take.

Fair point. Open to suggestions in future PRs for improving this!